### DISCUSSION TITLE
Having trouble importing SDA maflies into the program

### DISCUSSION THREAD

**Discussion by 16546489** [dyc3/steamguard-cli#215] [Q&A] [answered]
I'm using a linux ubuntu distribution.
I'm not sure how to properly import the sda maflies and manifest.json into the program.
I tried deleting the manifest and adding it after I imported the maflies I get an error like this
/# steamguard import --sda
INFO - reading manifest from /root/.config/steamguard-cli/maFiles
ERROR - Did not find manifest in /root/.config/steamguard-cli/maFiles
Would you like to create a manifest in /root/.config/steamguard-cli/maFiles ? [Yn] Y
INFO - Saving manifest and accounts...
INFO - Saving manifest and accounts...
steamguard
INFO - reading manifest from /root/.config/steamguard-cli/maFiles
ERROR - Could not load accounts: Could not find an entry in the manifest for this account. Check your spelling.
ERROR - Could not find an entry in the manifest for this account. Check your spelling.
and when I check the manifest
there is no accounts in it.

**Comment by 16546489** [comment]
I have also tried using this steamguard import --sda ./<example>.maFile
error: Found argument './<example>.maFile' which wasn't expected, or isn't valid in this context.
I have registered an account with the program and I noticed the maflies are different and the manifest.json as well so that might cause some unexpected behavior. I'm not sure how to properly make it work since sda seems to be outdated but I have multiple account on it.

**Comment by dyc3** [comment, marked as answer]
Try following the instructions here: https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md#importing-existing-mafiles-from-steam-desktop-authenticator
